server/views/pages/partials/ersedCannotHappenNotificationBanner.njk

```diff
@@ -1,9 +1,9 @@
 {%- from "moj/components/banner/macro.njk" import mojBanner -%}
 
-{% if model.ersedNotApplicableDueToDtoLaterThanCrd() %}
+{% if model.calculationBreakdown.ersedNotApplicableDueToDtoLaterThanCrd %}
```

Comment:
I don't think calculation breakdown is always available so maybe this is unsafe? I'm not 100% sure about in practice but it's nullable in "model"

Reply:
Added null check and also additional test case GET /calculation/:nomsId/summary/:calculationRequestId should not error and also not display ERSED Not applicable banner when calculationBreakdown is null in calculationRoutes.test.ts